app/app/vhosts/edit/route.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default Ember.Route.extend({
+  title: function(){
+    let app = this.modelFor('app');
+
+    return `Edit ${this.currentModel.get('virtualDomain')} - ${app.get('handle')}`;
```

Comment:
True but if I understand it the title logic will also append the stack name, which we don't want.